ci-operator/step-registry/openstack/provision/sriov-operator/openstack-provision-sriov-operator-commands.sh

```diff
@@ -92,6 +92,7 @@ if [ -n "${is_dev_version:-}" ]; then
     export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=quay.io/openshift/origin-sriov-network-config-daemon:${oc_version}
     export SRIOV_NETWORK_WEBHOOK_IMAGE=quay.io/openshift/origin-sriov-network-webhook:${oc_version}
     export SRIOV_NETWORK_OPERATOR_IMAGE=quay.io/openshift/origin-sriov-network-operator:${oc_version}
+    export OVS_CNI_IMAGE=""
```

Comment:
Could you perhaps add a comment for why we set an empty environment variable?
Also, is it better to set the var to empty string, or to unset the var entirely?

Reply:
It seems to me that a better alternative would be to set a default value (to empty string) in the [sriov-network-operator's hack/env.sh](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/4b3a309/hack/env.sh#L15).

Reply:
Alternative proposed at https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/683